build.sh

```diff
@@ -90,6 +96,7 @@ CMAKE_CMD="cmake -DBUILD_UNIT_TEST=${BUILD_UNITTEST} \
 -DCMAKE_CUDA_COMPILER=${CUDA_COMPILER} \
 -DENABLE_PROFILING=${SUPPORT_PROFILING} \
 -DKNOWHERE_GPU_VERSION=${SUPPORT_GPU} \
+-DKNOWHERE_SUPPORT_DISKANN=${SUPPORT_DISKANN} \
```

Comment:
Plz let me know if you have any other thoughts. I think we don't need this switch. Since:
1. `DiskANN` support is by default just like `FAISS`, `HNSW`, etc.
2. This PR will be merged to `support_diskann` branch instead of the main. So we don't need to worry about any damage for now.